app/models/hearing.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
 class Hearing
-  attr_reader :data
+  attr_reader :data, :skip_events
 
   delegate :blank?, to: :data
 
-  def initialize(data)
+  def initialize(data, skip_events: false)
```

Comment:
is this a double negation? 🤔  **skip** event **false**

What about using direct logic `include_events: true`?
also "include" sounds better